addon/models/node.js

```diff
@@ -38,6 +38,7 @@ export default OsfModel.extend(FileItemMixin, {
     dateCreated: DS.attr('date'),
     dateModified: DS.attr('date'),
 
+    nodeLicense: DS.attr(),
```

Comment:
Yes, but there they are called licenseRecord, since that makes more sense in the general concept of a license, when attached to something that is not a node.